static/js/compose_error.ts

```diff
@@ -1,27 +1,83 @@
 import $ from "jquery";
```

Comment:
I wonder if this file should be called `compose_banner.ts`, since that seems like a more descriptive name now

Reply:
I think that'd be a reasonable name change; if you want to do that, we should do the rename in its own commit; maybe we can decide when this branch is merged and just do the rename as a follow-up if we like the idea.